RELEASING.md

```diff
@@ -0,0 +1,58 @@
+# How to make a libxkbcommon release
+
+### Prerequisites
+
+- Have write access to xkbcommon Git repositories.
+- Be subscribed to the [wayland-devel](https://lists.freedesktop.org/mailman/listinfo/wayland-devel) mailing list.
+
+### Steps
+
+#### Prepare the release
+
+- [ ] Update the `NEWS.md` file for the release, following [the corresponding instructions](changes/README.md).
+
+- [ ] Bump the `version` in `meson.build`.
+
+- [ ] Run `meson dist -C build` to make sure the release is good to go.
+
+- [ ] Commit `git commit -m 'Bump version to MAJOR.MINOR.PATCH'`.
+
+- [ ] Tag `git tag --anotated -m xkbcommon-<MAJOR.MINOR.PATCH> xkbcommon-<MAJOR.MINOR.PATCH>`.
+
+- [ ] Push `git push --follow-tags`.
+
+#### Send announcement email to wayland-devel
+
+- [ ] Send an email to the wayland-devel@lists.freedesktop.org mailing list, using this template:
+
+```
+Subject: [ANNOUNCE] libxkbcommon MAJOR.MINOR.PATCH
+
+<NEWS & comments for this release>
+
+Git tag:
+--------
+
+git tag: xkbcommon-<MAJOR.MINOR.PATCH>
+git commit: <git commit sha>
+
+<YOUR NAME>
+```
+
+#### Update website
+
+- [ ] Pull the latest [website repository](https://github.com/xkbcommon/website).
+
+- [ ] Add the doc for the release: `cp -r <xkbommon>/build/html doc/<MAJOR.MINOR.PATCH>`.
```

Comment:
We should define a range of versions for Doxygen. It is not that reliable, things (e.g. links) got broken/repaired multiple times.

IIRC our runners use a version that is bit old and there are some issues in the display.

Reply:
That would be the best way indeed. But the issue is that Doxygen breaks things regularly (they do not have much regression test suite), so we need some manual control.

E.g. currently I have the latest version (1.12.0) and I get these warnings:
```
...
home/user/Repositories/libxkbcommon/include/xkbcommon/xkbcommon.h:1775: warning: unable to resolve reference to 'virtual-modifier-def' for \ref command
/home/user/Repositories/libxkbcommon/include/xkbcommon/xkbcommon.h:1797: warning: unable to resolve reference to 'virtual-modifier-def' for \ref command
...
```

This is fixed in current master, which we could use if generating the doc manually.

All in all I would like to drop Doxygen, but it seems there is no replacement that’s as complete, easy to install and free.

Reply:
> E.g. currently I have the latest version (1.12.0) and I get these warnings:

Indeed I have seen those.

> All in all I would like to drop Doxygen, but it seems there is no replacement that’s as complete, easy to install and free.

In the mean time, let's try to publish the docs with a good build and hope they release a new version soon...